src/core/SampleBuffer.cpp

```diff
@@ -792,6 +792,11 @@ bool SampleBuffer::play( sampleFrame * _ab, handleState * _state,
 	{
 		_ab[i][0] *= m_amplification;
 		_ab[i][1] *= m_amplification;
+        if (m_fading)
+        {
+            _ab[i][0] *= fadefunc(play_frame+i);
+            _ab[i][1] *= fadefunc(play_frame+i);
+		}
```

Comment:
```suggestion
	if (m_fading)
	{
		_ab[i][0] *= fadefunc(play_frame+i);
		_ab[i][1] *= fadefunc(play_frame+i);
	}
```
Converts spaces to tabs and fixes indentation on last bracket.